test/flycheck-test.el

```diff
@@ -3473,6 +3473,14 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
      '(9 9 warning "Ignored `error` returned from `os.Stat(\"enoent\")`"
          :checker go-errcheck))))
 
+(flycheck-ert-def-checker-test go-errcheck go nil
```

Comment:
Sorry, I've been rather busy this weekend. I'll get to it in the next 2 days and ping you when I've done so.